deploy/runtime/ui/Operating Costs Land Lease.json

```diff
@@ -463,19 +463,19 @@
                 },
                 "X": {
                     "Type": 3.0,
-                    "Integer": 276.0
+                    "Integer": 273.0
```

Comment:
![image](https://github.com/NREL/SAM/assets/54449384/4010b030-31ca-41f0-b1e1-da7663febbbb)
Not sure if this will clip outside of Windows?